Rasa_Bot/actions/helper.py

```diff
@@ -1140,13 +1140,13 @@ def make_step_overview(date_array: List[str], step_array: List[int], step_goal:
          'steps': step_array,
          'goals': step_goal})
 
-    data['goal_achieved'] = data['steps'] >= data['goals']
+    data['goal_achieved'] = (data['steps'] >= 0.95*data['goals'])*1 + (data['steps'] >= data['goals'])*1
```

Comment:
```suggestion
    data['goal_achieved'] = (data['steps'] >= 0.95*data['goals']) + (data['steps'] >= data['goals'])*2
```
If I get it right, it should be like this, right?

Reply:
Nope, note that, if: _data['steps'] > data['goals'],_ it also satisfies _data['steps'] > 0.95*data['goals']_, meaning we will have 1+1 = 2

Reply:
Oh yes, I see